installers/linux/deb-scripts/prerm

```diff
@@ -1,7 +1,5 @@
 #!/bin/sh
 
-set -e
```

Comment:
I removed `set -e` from all of the linux scripts. It made the install / uninstall code very fragile.  For example, in 

```
grep "^$USERNAME:" /etc/passwd
if [ $? -eq 0 ]
...
```

The `postinst` script aborted immediately after the grep if the account didn't already exist.

In the `service_stop` script, when trying to stop and disable the service, the script aborted if the service wasn't registered yet.  This caused `prerm` to abort too.

Both of these caused the `dpkg -i` to fail hard and that put the package in a very weird state. Something like, "your package is very broken. please repair it before removing it."  And then "Cannot repair package, it is very broken."

Also, various `-force...` options had no effect, since the repair process re-ran the `prerm` and `postinst` scripts again.  And the `dpkg` diagnostics were not very helpful (er, cryptic).

So I removed all of the `set -e` settings.  There may be better ways to do this -- or I could add more error checking to various commands (where I'm not immediately using the $?), but this got me unstuck.